### PR TITLE
MRG: Updates to the installation instructions & small tweaks to other parts of the documentation

### PR DESCRIPTION
**Pull request by hoechenberger**
- The command line meant to run the installer on Linux now comes with a copy button
- Switch from `.. code-block:: console` to `... bash` everywhere, as `Pygments`, which is reliable for the syntax highlighting, doesn't know any `console`
- Now always show the copy button, not only on hovering above the code blocks. I believe this makes this functionality easier to discover.
- Add manual installation instructions for Apple Silicon.

### CODE REVIEW

File: doc/install/installers_platform_selector.html [hunk 1]
```diff
@@ -53,7 +53,7 @@ <h5 class="m-0 font-weight-bold">
         </a>
         <p class="small"><strong>Supported platforms:</strong> Ubuntu 18.04 (Bionic Beaver) and newer
         </p>
-        <p>Run the installer in a terminal via: <code>sh ./MNE-Python-1.0.2_0-Linux.sh</code></p>
+        <p>Run the installer in a terminal via: <div class="highlight"><pre>sh ./MNE-Python-1.0.2_0-Linux.sh</pre></div></p>
```
Comment (hoechenberger) [inline doc/install/installers_platform_selector.html:56]:
```suggestion
        <p>Run the installer in a terminal via: <div class="highlight highlight-bash"><pre>sh ./MNE-Python-1.0.2_0-Linux.sh</pre></div></p>
```

**Comment by larsoner** [comment]
> Switch from .. code-block:: console to ... bash everywhere, as Pygments, which is reliable for the syntax highlighting, doesn't know any console

Did you consider bash session instead? https://pygments.org/languages/ It seems like a more appropriate name for our "BASH prompt" type of examples, but I haven't tested. I can test locally if needed...

**Comment by hoechenberger** [comment]
Rendered docs:

https://output.circle-artifacts.com/output/job/779f2502-8921-410e-b161-122126b094b5/artifacts/0/dev/install/manual_install.html

**Comment by hoechenberger** [comment]
> Did you consider bash session instead?

No, is there a difference?

**Comment by hoechenberger** [comment]
> > Did you consider bash session instead?
> 
> No, is there a difference?

Ah it seems to add a prompt.

I'd rather we didn't do that tbh, it's confusing because the "example prompt" never matches what the users actually see on their own system…

**Comment by larsoner** [comment]
> I'd rather we didn't do that tbh, it's confusing because the "example prompt" never matches what the users actually see on their own system…

I think having the `$` is informative/helpful

**Comment by hoechenberger** [comment]
> I think having the `$` is informative/helpful

I'd argue that's only the case for those "who know"; and those who know already know and don't need this.

For all others, it's confusing – especially on Windows.

**Comment by hoechenberger** [comment]
Alright, but we shouldn't have this in the installation instructions, where we lump together the prompts for several operating systems that DO work in all supported shells; e.g. here, the command to create a conda env on macOS, Windows, and Linux:

https://output.circle-artifacts.com/output/job/779f2502-8921-410e-b161-122126b094b5/artifacts/0/dev/install/manual_install.html

**Comment by larsoner** [comment]
> Alright, but we shouldn't have this in the installation instructions, where we lump together the prompts for several operating systems that DO work in all supported shells; e.g. here, the command to create a conda env on macOS, Windows, and Linux:

I can live with no `$` there. For consistency maybe we should use the same phrasing "the terminal or an Anaconda Prompt" as in https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html

**Comment by hoechenberger** [comment]
Sounds good to me! Do you want to do these changes?

**Comment by larsoner** [comment]
No, feel free since you're working on this stuff

**Comment by hoechenberger** [comment]
Ouch, turns out Pygments **does** support `console` – it's an alias for `shell-session`. 

The reason why this didn't render correctly in some places was …

… because it expects a leading `$` (or any other prompt – some auto detection seems to be going on there)

So I'll revert 283a181 and try to fix the occasions where things don't seem to be rendered correctly.

**Comment by hoechenberger** [comment]
Ok … let's hope this comes back looking nicely.

Turns out you're getting it your way, @larsoner  ;)

**Comment by hoechenberger** [comment]
@larsoner Should be good to merge once CI is done!!

**Comment by hoechenberger** [comment]
Rendered install docs:
https://output.circle-artifacts.com/output/job/e881fc93-74c8-4421-a3eb-47b034b886d3/artifacts/0/dev/install/manual_install.html

**Comment by larsoner** [comment]
Thanks @hoechenberger !

**Comment by hoechenberger** [comment]
@larsoner Do you think we can backport this?

**Comment by larsoner** [comment]
Feel free, especially since it looks like #10580 was backported and this PR has fixes for some of the code blocks changed there

**Pull request merged**